cfme/tests/automate/test_domain.py

```diff
@@ -564,3 +566,87 @@ def test_redhat_domain_sync_after_upgrade(temp_appliance_preconfig, file_name):
         temp_appliance_preconfig.db.restore_database(
             db_path, is_major=bool(temp_appliance_preconfig.version > "5.11")
         )
+
+
+@pytest.fixture
+def custom_domain(custom_instance):
+    """This fixture creates dastastore setup and updates the name and description of domain. So that
+       the domain with same name can be imported successfully.
+    """
+    instance = custom_instance(ruby_code=None)
+    # Domain name and description are updated because we are importing domain with same name via
+    # import datastore file
+    instance.domain.update({"name": "bz_1752875", "description": "bz_1752875"})
```

Comment:
Do you think adding some random characters at the end would help? So that there is no overlap, or multiple domains with same name clashing, failing to create etc?

Reply:
Yes, I can understand but it is not possible. As we need same domain name and description as commented which should match with domain we are importing.

Reply:
Ok. Thanks.